cmd/rule-evaluator/main.go

```diff
@@ -131,22 +139,19 @@ func main() {
 
 	extraArgs, err := exportsetup.ExtraArgs()
 	if err != nil {
-		//nolint:errcheck
-		level.Error(logger).Log("msg", "Error parsing commandline arguments", "err", err)
+		_ = level.Error(logger).Log("msg", "Error parsing commandline arguments", "err", err)
```

Comment:
Question: Is throwing away the return better than nolint? I honestly don't have a strong preference, just curious if you have thoughts.

Reply:
I typically have a difficult time remembering the nolint warning so I find this easier. Less boilerplate, clear and concise, and future-proof from other linters (when I was in the Java world, there were overlapping rules between linters and maybe one day could happen in Golang, who knows).